azkaban-common/src/main/java/azkaban/executor/container/ContainerImplUtils.java

```diff
@@ -61,4 +63,18 @@ private static void populateJobTypeForFlow(final ExecutableNode node, Set<String
       jobTypes.add(node.getType());
     }
   }
+
+  /**
+   * Return the int mapping between 1 to 100 for a given flow name
+   * @param flow
+   * @return
+   */
+  public static int getFlowNameHashValMapping(ExecutableFlow flow) {
+    // Flow name is <projectName>.<flowId>
+    String flowName = flow.getFlowName();
+    byte[] flowNameBytes = flowName.getBytes(StandardCharsets.UTF_8);
+    // To be utilized for flow deterministic ramp-up
+    int flowNameHashVal = MurmurHash3.hash32x86(flowNameBytes);
+    return flowNameHashVal % 100 + 1;
```

Comment:
Should it be something like `(double)flowNameHashVal / MAX_HASH_VALUE` 
where MAX_HASH_VALUE will be UINT32_MAX for 32 bit hash. 
This will have the advantage of using the entire hashed value without relying on only some of the least-significant-bits and should not have any extra cost.